include/experimental/__p0009_bits/accessor_basic.hpp

```diff
@@ -52,12 +52,23 @@ namespace experimental {
 
 template <class ElementType>
 struct accessor_basic {
-  
+
   using offset_policy = accessor_basic;
   using element_type = ElementType;
   using reference = ElementType&;
   using pointer = ElementType*;
 
+  constexpr accessor_basic() noexcept = default;
+
+  MDSPAN_TEMPLATE_REQUIRES(
+    class OtherElementType,
+    /* requires */ (
+      _MDSPAN_TRAIT(is_convertible, typename accessor_basic<OtherElementType>::pointer, pointer)
+    )
+  )
+  MDSPAN_INLINE_FUNCTION
+  constexpr accessor_basic(accessor_basic<OtherElementType>) {}
```

Comment:
@nliber should we declare it `explicit`?

Reply:
this way we don't need assign explicitly. But if Nevin thinks its better.

Reply:
I think implicit is correct.

Note:  we also need the constraint:

```
  MDSPAN_TEMPLATE_REQUIRES(
    class OtherElementType,
    /* requires */ (
      _MDSPAN_TRAIT(is_convertible, OtherElementType, ElementType)
    )   
  )
  MDSPAN_INLINE_FUNCTION
  constexpr accessor_basic(accessor_basic<OtherElementType>) noexcept {}

```

(I already have that implemented as part of the changes to `mdspan` to match [P0009R12](http://wg21.link/P0009R12), if you want to wait for that P/R later this week).